ui/components/app/selected-account/selected-account.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import SelectedAccount from '.';
+
+export default {
+  title: 'Components/App/SelectedAccount',
+  argTypes: {
+    copied: {
+      control: 'boolean',
+    },
+  },
+};
+
+export const DefaultStory = (args) => <SelectedAccount {...args} />;
+
+DefaultStory.storyName = 'Default';
```

Comment:
Adding story for easier future review

<img width="1438" alt="Screenshot 2023-02-16 at 4 01 17 PM" src="https://user-images.githubusercontent.com/8112138/219527153-57fee77e-72c4-40f7-9345-237b59aeced2.png">